File: delete_duplicates.java
public class Solution {
    public ListNode deleteDuplicates(ListNode head) {
        if(head==null)
            return head;
        ListNode tmp = head;
        while(tmp!=null){
            if(tmp.next!=null && tmp.val == tmp.next.val){
                tmp.next = tmp.next.next;
            }
            else
                tmp = tmp.next;
        }
        return head;
    }
}

File: delete_duplicates.py
class Solution(object):
    def deleteDuplicates(self, head):
        """
        :type head: ListNode
        :rtype: ListNode
        """
        if head is None:
            return None
        tmp = head
        while tmp is not None:
            if tmp.next is not None and tmp.val == tmp.next.val:
                tmp.next = tmp.next.next
            else:
                tmp = tmp.next
        return head
            